exporter/datadogexporter/examples/collector.yaml

```
@@ -48,6 +48,25 @@ receivers:
   #       static_configs:
   #       - targets: ['0.0.0.0:8888']
 
+  ## Read more about filelog receiver https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver .
+  filelog:
+    include_file_path: true
+    poll_interval: 500ms
+    include:
+      ## Logs will be collected from the below file path. Please ensure it is correct or edit as needed.
+      - /var/log/**/*example*/*.log
+     ## It is essential to extract trace_id and span_id from the logs in order to ensure that logs / traces correlation works.
+     ## For JSON logs, json_parser will parse the body into attributes, while trace_parser will extract the trace_id and span_id.
```

This is a commented out block. But can you please make it like the other commented out blocks and add a space between the two hash tags? The idea is that you remove the first two characters from the entire block and then it is usable immediately.
```suggestion
     # # It is essential to extract the trace_id and span_id from the logs in order to ensure that logs / traces correlation works.
     # # For JSON logs, json_parser will parse the body into attributes, while trace_parser will extract the trace_id and span_id.
```